src/register/register.ts

```diff
@@ -30,9 +32,20 @@ export class Register {
    */
   private static registers: { [key: string]: IRegisterContent } = {
     '"': { text: "", registerMode: RegisterMode.CharacterWise },
-    '*': { text: "", registerMode: RegisterMode.CharacterWise }
+    '*': { text: "", registerMode: RegisterMode.CharacterWise, isClipboardRegister: true },
+    '+': { text: "", registerMode: RegisterMode.CharacterWise, isClipboardRegister: true }
   };
 
+  public static isClipboardRegister(registerName: string): boolean {
+    const register = Register.registers[registerName];
+
+    if (register && register.isClipboardRegister) {
```

Comment:
Ohhhhh. It's because it's an optional property, which has type `boolean | undefined`, which we are returning directly. That makes sense... in a weird way. :) 

I guess TypeScript is pointing out that we did a dumb thing by saying the type was `isClipboardRegister?: boolean` since now we can't distinguish `undefined` from `false` in an easy way.

Best is probably to change the type to `isClipboardRegister: boolean`, and add it to all registers. Then this code will work and also our `undefined` vs `false` case will go away. 

TypeScript caught a (minor) bug! Yay, TypeScript!